parm/gfs/gfs_plots.yaml.orig

```diff
@@ -0,0 +1,588 @@
+# DA Monitoring source dictionary
```

Comment:
Do we need to commit this original file backup?

Reply:
Oh gosh no.  I must have been more tired than I thought yesterday.  I'll remove this from the PR.